torchbenchmark/models/hf_Falcon_7b/__init__.py

```diff
@@ -0,0 +1,10 @@
+from torchbenchmark.tasks import NLP
+from torchbenchmark.util.framework.huggingface.model_factory import HuggingFaceModel
+
+class Model(HuggingFaceModel):
+    task = NLP.LANGUAGE_MODELING
+    DEFAULT_TRAIN_BSIZE = 4
```

Comment:
Can we have a link to upstream code that reference the train batch size?

Reply:
The Falcon README shows the training batch size to be 2304 [link](https://huggingface.co/tiiuae/falcon-7b/blob/main/README.md#:~:text=Batch%20size,tokens%20ramp%2Dup)] - included link in comment. This seems prohibitive to run in testing suite though, so I kept the default size to 4.

Reply:
Thanks for making the reference! We should use the per-device batch size here. In the link it mentioned 2304 batch size on 384 A100 GPUs, so per-GPU batch size should be 6. Can you please update it to 6 and add the link ( https://huggingface.co/tiiuae/falcon-7b/blob/main/README.md#:~:text=Batch%20size,tokens%20ramp%2Dup) as the reference?

Reply:
Thanks for the clarification! Updated the size and the reference link.